### ISSUE TITLE
Do any of your repositories work with Dashboard 2?

### ISSUE THREAD

**Issue by riprock67** [hotNipi/node-red-contrib-ui-level#48]
Dear HotNipi -

I was wondering if any of your current repositories work with Dashboard 2, but especially the ui-level repository.  My sincere thanks,

RipRock

**Comment by hotNipi** [comment]
No. But I made couple of gauge prototypes you can find here https://discourse.nodered.org/t/dashboard-2-0-is-now-generally-available/84913/57?u=hotnipi

**Comment by riprock67** [comment]
Thanks!

I will give these a shot!  It is also nice to see a link to the discord where active questions are being entertained.

Scott
 
*************************************************************************************
Dr. F. Scott Anderson Tel: (720) 240-0127
Southwest Research Institute Ofc: (303) 546-9670
1050 Walnut St, Suite 300  Cell: (808) 224-4734
Boulder, CO 80302  Fax: (303) 546-9687
http://www.boulder.swri.edu/
*************************************************************************************
All opinions herein stated are mine, not those of SwRI or NASA.  Trust me.

**Comment by mecworks** [comment]
@hotNipi, any ETA on when there will be a level widget for dashboard 2.x?
